### PR TITLE
Translation 

### PR DESCRIPTION
**Pull request by Nathanael-Rayapin**
news translations about accessibility in other countries

### CODE REVIEW

**Comment by etsraphael** [comment]
You created the references but you're not using it. Look at this file : 

`/src/app/core/service/auth/auth.service.ts` the reference have to be linked there :)

File: src/app/core/service/translate/translate.service.ts [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
 import { TranslateService } from '@ngx-translate/core';
+import { BehaviorSubject } from 'rxjs';
 
 @Injectable({ providedIn: 'root' })
 export class TranslationService {
-  constructor(private translate: TranslateService) {}
+  constructor(public translate: TranslateService) {}
 
+  // Observable (current Snackbar data)
+  currentText = new BehaviorSubject<string>(null);
```
Comment (etsraphael) [inline src/app/core/service/translate/translate.service.ts]: Actually, you don't have to go with your `currentText` value, we can save a const if you just call your snackbar with your string

File: src/app/core/service/auth/auth.service.ts [hunk 2]
```diff
@@ -29,7 +31,8 @@ export class AuthService {
   login(): void | MatSnackBarRef<TextOnlySnackBar> {
     if (environment.production) {
       return this.snackBarService.openSnackBar(
-        'Not available in your country yet'
+        this.translate.currentText.getValue(),
```
Comment (etsraphael) [inline src/app/core/service/auth/auth.service.ts]: Why ```this.translate.currentText.getValue()``` ? Just pour your string value here

File: src/app/core/service/translate/translate.service.ts [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
   }
 
+  // Set new languages
   setNewLang(lang: string): void {
     this.translate.use(lang);
   }
+
+  // Display a Snackbar with current lang of interface User
+  translateInCurrentLang(): void {
+    this.translate
```
Comment (etsraphael) [inline src/app/core/service/translate/translate.service.ts]: this.translate.instant('my.value.to.translate') is your solution here

File: src/app/views/on-boarding/on-boarding-main/on-boarding-main.component.ts [hunk 4]
```diff
@@ -74,7 +74,8 @@ export class OnBoardingMainComponent {
 
   notAvailableInYourCountry(): MatSnackBarRef<TextOnlySnackBar> {
     return this.snackBarService.openSnackBar(
-      'Not available in your country yet'
+      this.translate.currentText.getValue(),
```
Comment (etsraphael) [inline src/app/views/on-boarding/on-boarding-main/on-boarding-main.component.ts]: To simplify

File: src/assets/i18n/en.json [hunk 5]
```diff
@@ -43,14 +43,21 @@
     "space-token": {
       "text": "Thanks to cryptocurrency, you will be able to stake and earn 'Undefined token' at the same time. You also be able to use 100% of the platform with full content available to a subscription. Wiin's certified Artist can be rewarded with monetary gifts. Join the Wiin's communtiy today and lets grow Zion's worth together ",
       "title": "Token",
-      "button-buy-token": "Buy 'Undefined token'"
+      "button-buy-token": "Buy in Pancake Swap"
```
Comment (etsraphael) [inline src/assets/i18n/en.json]: Maybe put an indicator here cause we can buy and Pancake Swap and Sushi Swap. So 2 references

File: src/app/core/service/translate/translate.service.ts [hunk 6]
```diff
@@ -3,13 +3,20 @@ import { TranslateService } from '@ngx-translate/core';
 
 @Injectable({ providedIn: 'root' })
 export class TranslationService {
-  constructor(private translate: TranslateService) {}
+  constructor(public translate: TranslateService) {}
```
Comment (etsraphael) [inline src/app/core/service/translate/translate.service.ts]: It should be private here

Comment (Nathanael-Rayapin) [reply src/app/core/service/translate/translate.service.ts]: done

File: src/assets/i18n/en.json (hunk 5 above)
Comment (Nathanael-Rayapin) [reply src/assets/i18n/en.json]: I added both, pancake and sushi

File: src/app/views/on-boarding/on-boarding-main/on-boarding-main.component.ts (hunk 4 above)
Comment (Nathanael-Rayapin) [reply src/app/views/on-boarding/on-boarding-main/on-boarding-main.component.ts]: resolve

File: src/app/core/service/translate/translate.service.ts (hunk 3 above)
Comment (Nathanael-Rayapin) [reply src/app/core/service/translate/translate.service.ts]: done

File: src/app/core/service/auth/auth.service.ts (hunk 2 above)
Comment (Nathanael-Rayapin) [reply src/app/core/service/auth/auth.service.ts]: done

File: src/app/core/service/translate/translate.service.ts (hunk 1 above)
Comment (Nathanael-Rayapin) [reply src/app/core/service/translate/translate.service.ts]: done

**Review by etsraphael — APPROVED** [review]
So good mate thanks!

**Pull request merged**